modules.d/35network-manager/nm-initrd.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 DefaultDependencies=no
-Wants=systemd-udev-settle.service
-After=systemd-udev-settle.service
+Wants=systemd-udev-trigger.service
+After=systemd-udev-trigger.service
```

Comment:
???

Reply:
Sorry, no idea. can you reject above change?

Reply:
> Sorry, no idea. can you reject above change?

no.... you have to get your git commit history right